### ISSUE TITLE
Report details-summary with NVDA+D

### ISSUE THREAD

**Issue by feerrenrut** [nvaccess/nvda#13380]

**Comment by lukaszgo1** [comment]
> `NVDA+d` will report a long description if it is available, this is rare.

I agree that long descriptions are very seldom seen in the wild, but `details-summary` are essentially non  existent. Are there any applications where they are indeed used?

**Comment by feerrenrut** [comment]
Yes, there will be more usages of this in the future, specifically in google docs/sheets. They won't start using it until it is supported in AT.

**Comment by feerrenrut** [comment]
After looking into this some more, I propose that we just re-assign the gesture directly to the report summary of details command. Due to the following reasons:
- The `longdesc` attribute is only valid on an `img` element.
- `longdesc` does not seem to be exposed in Chrome / edge, it continues to work in Firefox.
- `longdesc` is deprecated, mdn advises not using it and transitioning away from current uses.
- `longdesc` has an incompatible UX with aria details. The `longdesc` attribute contains a URL, the browser exposes an action, when triggered the URL is opened in a new tab. Aria details is expected to be read without leaving the page.

Anyone who still relies on `longdesc` support can assign an alternative gesture.
